Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -700,6 +700,7 @@ type HomeScreenStateProps =
   , canSendSuggestion :: Boolean
   , sheetState :: BottomSheetState
   , showDisabilityPopUp :: Boolean
+  , dummyCounter :: Int
```

Comment:
@ashishsingh101 do we need this variable?

Reply:
no